feature_layers/display-route-layer/src/main/java/com/esri/samples/display_route_layer/DisplayRouteLayerSample.java

```diff
@@ -0,0 +1,166 @@
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.scene.text.TextAlignment;
+import javafx.stage.Stage;
+
+
+public class DisplayRouteLayerSample extends Application {
+
+    private MapView mapView;
+    private PortalItem portalItem; // keep loadable in scope to avoid garbage collection
+
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Display Route Layer Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // authentication with an API key or named user is required to access basemap and other location services
+            String yourAPIKey = System.getProperty("apiKey");
+            ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+            // create a map with the topographical basemap style
+            ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+            // create a map view and set the map to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // set a viewpoint on the map view
+            mapView.setViewpoint(new Viewpoint(45.2281, -122.8309 + 0.2, 57e4));
+
+            //create a VBox
+            var vBox = new VBox();
+            vBox.setMaxSize(300, 400);
+            vBox.setPadding(new Insets(10.0));
+            vBox.setAlignment(Pos.TOP_CENTER);
+            vBox.setSpacing(10);
+            vBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0, 0, 0 , 0.7)"), CornerRadii.EMPTY, Insets.EMPTY)));
+
+            //create scrollPane
+            var scrollPane = new ScrollPane();
+            scrollPane.setMaxSize(300, 400);
+            scrollPane.setPadding(new Insets(10.0));
+
+            VBox directionsVBox = new VBox();
+            directionsVBox.setSpacing(10);
+            scrollPane.setContent(directionsVBox);
+            var headingLabel = new Label("Directions:");
+            headingLabel.setStyle("-fx-font-weight:bold; -fx-text-fill: white; -fx-font-size: 14;");
+            vBox.getChildren().addAll(headingLabel, scrollPane);
+
+            // create portal and portalItem
+            var portal = new Portal("https://www.arcgis.com/");
+            portalItem = new PortalItem(portal, "0e3c8e86b4544274b45ecb61c9f41336");
+            portalItem.loadAsync();
+            portalItem.addDoneLoadingListener(() -> {
+
+                if (portalItem.getLoadStatus() == LoadStatus.LOADED) {
+                    var featureCollection = new FeatureCollection(portalItem);
+                    featureCollection.loadAsync();
```

Comment:
```suggestion
                    // load the feature collection and access its tables
                    featureCollection.loadAsync();
```
Because this is a sample, we need to add helpful comments for users to follow